frontend-react/src/components/ManagePublicKey/ManagePublicKeyChooseSender.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+import { Form, Dropdown, Button, FormGroup } from "@trussworks/react-uswds";
+
+import { UseOrganizationSenders } from "../../hooks/UseOrganizationSenders";
+import Spinner from "../Spinner";
+
+type Props = {
+    onSenderSelect: any; //What type should this be???
+};
+
+export function ManagePublicKeyChooseSender({ onSenderSelect }: Props) {
+    const { isLoading, senders } = UseOrganizationSenders();
+    const [selectedSender, setSelectedSender] = useState(
+        senders?.length === 1 ? senders[0].name : ""
+    );
+
+    if (senders?.length === 1) {
+        onSenderSelect(selectedSender);
+    }
+
+    function handleSubmit() {
+        onSenderSelect(selectedSender);
+    }
+
+    return (
+        <>
+            {isLoading && <Spinner message="Processing file..." />}
+            {!isLoading && senders && (
+                <Form name="senderSelect" onSubmit={handleSubmit}>
```

Comment:
Let's be consistent with our names.  We're using a pattern of lowercase and hyphenated names already: `sender-select`